server/database/queries/offers/saved-offers.js

```diff
@@ -0,0 +1,12 @@
+const dbconnection = require('../../config/db_connection');
+
+const addSavedOffers = (savedOfferInfo) => {
+  const queryValues = Object.values(savedOfferInfo);
```

Comment:
@Fatmasiam I think it's possible that you won't always get the query values in the order that you expect, so it might be safer to be more specific:
```
const { memberId, offerId } = savedOfferInfo
const sql = {
  text: `INSERT INTO ......
  values: [ memberId, offerId ]
}
```

This also prevents bad users from posting extra fields that you don't want them to be able to post to, if saved_offer had more fields (like if it had a "created_at" timestamp, you probably don't want to let users POST and overwrite that timestamp)